Project1/Swords.h

```diff
@@ -4,20 +4,18 @@
 
 class BronzeSword : public Weapon {
 private:
-	static const std::string _name;
-	
+	inline static std::shared_ptr<WeaponInfo> _shared_info = std::make_shared<WeaponInfo>(
+		"Bronze Sword",
+		WeaponType::values::SWD,
+		std::array<int, CombatStats::size>{ 5, 90, 0, 5, 0 }
+	);
 public:
-	BronzeSword() = default;
+	BronzeSword();
```

Comment:
So, since we won't have be coding a class for each specific weapon, should we remake this into Sword, like the filename suggests? The default constructor for sword can still make a bronze sword.

Reply:
Yes, we can move the bronze sword info into Sword (I think Weapon, actually) and remove these leaf classes.

Reply:
That's what I thought, but I would have been somewhat OK (not happy) with each WeaponType having a class.

Reply:
All the interesting ones probably will need their own, but no sense in doing that as a rule.